scripts/localnet-blocks-test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+CNT=0
```

Comment:
let's add a newline here. otherwise it's easy to mistake `CTN` for yet another ENV variable (`0` vs `$0`)